samples/bluetooth/fast_pair/locator_tag/configuration/pm_static_nrf52833dk_nrf52833_release.yml

```diff
@@ -0,0 +1,55 @@
+mcuboot:
+  address: 0x0
+  region: flash_primary
+  size: 0x4000
+
+mcuboot_primary:
+  address: 0x4000
+  orig_span: &id001
+  - mcuboot_pad
+  - app
+  region: flash_primary
+  size: 0x3c000
+  span: *id001
+mcuboot_pad:
+  address: 0x4000
+  region: flash_primary
+  size: 0x200
+app:
+  address: 0x4200
+  region: flash_primary
+  size: 0x3be00
+mcuboot_primary_app:
+  address: 0x4200
+  orig_span: &id002
+  - app
+  region: flash_primary
+  size: 0x3be00
+  span: *id002
+
+mcuboot_secondary:
+  address: 0x40000
+  orig_span: &id003
+  - mcuboot_secondary_pad
+  - mcuboot_secondary_app
+  region: flash_primary
+  size: 0x3c000
+  span: *id003
+mcuboot_secondary_pad:
+  region: flash_primary
+  address: 0x40000
+  size: 0x200
+mcuboot_secondary_app:
+  region: flash_primary
+  address: 0x40200
+  size: 0x3be00
+
+bt_fast_pair:
+  address: 0x7c000
+  region: flash_primary
+  size: 0x1000
+
+settings_storage:
+  address: 0x7d000
+  region: flash_primary
+  size: 0x3000
```

Comment:
I think we may consider to use `0x5000` for settings here. Locator tag periodically stores new clock value, which may lead to NVM wear off. What do you think @kapi-no, @mkapala-nordic ?

Reply:
Nope, I would keep the default size `0x2000` here.